### ISSUE TITLE
add an example dataset

### ISSUE THREAD

**Issue by lawremi** [waldronlab/MultiAssayExperiment#200]
There is complicated code in the examples and vignette for constructing a toy MAE, and there is a `hnscSample.rds` file that depends on the RaggedExperiment package. Would be nice to have a simple object for examples and testing in the `data/` directory, like `sample.ExpressionSet` in Biobase. Ideally it would be small but still real enough to get meaningful results from modeling, maybe just `hnscSample` without the extra dependency.

**Comment by lwaldron** [comment]
A nice ACC example dataset attached. It is based on the TCGA ACC dataset, providing RNA-seq, GISTIC recurrent copy number lesions, RPPA proteins, and mutations (0 for no or silent mutations, 1 for non-silent mutations). Metadata provides some source information. Marcel, would you add to the data/ directory? 

```
> miniacc
A MultiAssayExperiment object of 4 listed
 experiments with user-defined names and respective classes. 
 Containing an ExperimentList class object of length 4: 
 [1] RNASeq2GeneNorm: ExpressionSet with 187 rows and 79 columns 
 [2] gistict: SummarizedExperiment with 187 rows and 90 columns 
 [3] RPPAArray: ExpressionSet with 33 rows and 46 columns 
 [4] Mutations: matrix with 94 rows and 90 columns 
Features: 
 experiments() - obtain the ExperimentList instance 
 colData() - the primary/phenotype DataFrame 
 sampleMap() - the sample availability DataFrame 
 `$`, `[`, `[[` - extract colData columns, subset, or experiment 
 *Format() - convert ExperimentList into a long or wide DataFrame 
 assays() - convert ExperimentList to a list of rectangular matrices
> metadata(miniacc)
$title
[1] "Comprehensive Pan-Genomic Characterization of Adrenocortical Carcinoma"
$PMID
[1] "27165744"
$sourceURL
[1] "http://s3.amazonaws.com/multiassayexperiments/accMAEO.rds"
$RPPAfeatureDataURL
[1] "http://genomeportal.stanford.edu/pan-tcga/show_target_selection_file?filename=Allprotein.txt"
$colDataExtrasURL
[1] "http://www.cell.com/cms/attachment/2062093088/2063584534/mmc3.xlsx"
> 
```

**Comment by lwaldron** [comment]
Update - I added miRNA-seq data to this MAE, to provide an example of where network or mapping support would be useful. Rows without at least 5 counts in at least 5 samples are removed. I also added the `readLines()` R code to `metadata(miniacc)$Rcode`, noted the miRNA filtering in `abstract(miniacc[["miRNASeqGene"]])`. Protein annotations are still provided in `featureData(miniacc[["RPPAArray"]])`. Note that RNA-seq, GISTIC copy number, and mutations have been subset to include only genes corresponding to proteins on the RPPA array.

[miniacc.zip](https://github.com/waldronlab/MultiAssayExperiment/files/1002164/miniacc.zip)
